cmplxfoil/CMPLXFOIL.py

```diff
@@ -48,16 +49,16 @@ class is to provide an easy to use wrapper for xfoil for intergration
 # If the matplotlib import worked, try niceplots
 if plt:
     try:
-        import niceplots as nice
+        import niceplots
```

Comment:
Can't you keep this as `import niceplots as nice`? You shouldn't need to rename variables here. But also, I prefer the full name `niceplots` because `nice` may be too generic of a variable name to be defined at the module top level.